iree/test/e2e/linalg_ext_ops/BUILD

```diff
@@ -81,6 +106,8 @@ iree_check_single_backend_test_suite(
             # TODO(antiagainst): scan fails on Adreno GPUs due to driver bug.
             # Re-enable this once we have new devices with up-to-date drivers.
             "scan.mlir",
+            "linalg_transform.mlir",
+            "linalg_transform_spec.mlir",
```

Comment:
I still think putting passing tests in exclude next to failing tests is confusing and that could be fixed by either

1. Adding some comments
2. Creating a `LIT_TEST_FILES` variable and using `exclude = LIT_TEST_FILES + [`
3. Moving linalg_transform tests to `iree/test/e2e/linalg_transform/`

I think I'd prefer 3, to avoid too much build magic.